src/test/java/org/fundacionjala/coding/yury/InnerContentTest.java

```diff
@@ -0,0 +1,42 @@
+package test.java.org.fundacionjala.coding.yury;
+
+import main.java.org.fundacionjala.coding.yury.InnerContent;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by YuryOrtuno on 5/25/2017.
+ */
+public class InnerContentTest {
+    /**
+     *
+     */
+    @Test
+    public void innerOneWord() {
+        InnerContent innerContent = new InnerContent();
+        assertEquals("srot", innerContent.innerWord("sort"));
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void innerTwoWord() {
+        InnerContent innerContent = new InnerContent();
+        assertEquals("inner ctonnet", innerContent.innerWord("inner content"));
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void innerSentence() {
+        InnerContent innerContent = new InnerContent();
```

Comment:
You are repeating this code in all your tests.
Please put them in @before method or convert it to a static method.